### ISSUE TITLE
How to use flopsscaling - it seems not to work

### ISSUE THREAD

**Issue by idevcde** [JuliaPerf/BandwidthBenchmark.jl#6]
Hi,

I can not get flopsscaling working. I am receiving:

```
julia> BandwidthBenchmark.flopsscaling()
ERROR: UndefVarError: flopsscaling not defined
Stacktrace:
 [1] getproperty(x::Module, f::Symbol)
   @ Base ./Base.jl:35
 [2] top-level scope
   @ REPL[25]:1
```

I am on `Julia Version 1.7.0-rc2 (2021-10-20)` and `BandwidthBenchmark v0.1.0`.

How to correctly use it? Any comment / advice, pls?

**Comment by idevcde** [comment]
Ok, I think I tracked it. There seem to be a spelling in the code `flopscaling()` or documentation` flopsscaling()`: 

Please see:

[Code](https://github.com/JuliaPerf/BandwidthBenchmark.jl/blob/7dbac00a319965faaa2d9205e6d96905a6864146/src/bwscaling.jl#L21):

[Documentation](https://github.com/JuliaPerf/BandwidthBenchmark.jl/blob/7dbac00a319965faaa2d9205e6d96905a6864146/src/bwscaling.jl#L21):

**Comment by giordano** [comment]
Addressed by #8.
